js/Features.js

```diff
@@ -46,6 +46,13 @@ var Features = function (config) {
         );
     }
 
+    if (semver.gte(config.apiVersion, "1.21.0")) {
+        features.push(
+            {bit: 25, group: 'rxMode', mode: 'group', name: 'RX_SPI', haveTip: true},
```

Comment:
Just going with the flow here. The configurator is not the nicest bit of code I've seen...